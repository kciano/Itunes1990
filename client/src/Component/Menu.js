import React from 'react';
import Navbar from 'react-bootstrap/Navbar'
import Nav from 'react-bootstrap/Nav'

const Menu = () => {

  
    return (
        <div>
            <Navbar bg="dark" variant="dark">
                <Navbar.Brand href="/">Itunes Search</Navbar.Brand>
                    <Nav className="mr-auto">
            
                    
                </Nav>
            </Navbar>
        </div>
    );
};

export default Menu;