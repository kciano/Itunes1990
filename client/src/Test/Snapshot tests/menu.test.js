import React from 'react';
import Menu from '../../Component/Menu';
import renderer from 'react-test-renderer';


test("Main snapshot test", () => {
    const component = renderer.create(
        <Menu />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})