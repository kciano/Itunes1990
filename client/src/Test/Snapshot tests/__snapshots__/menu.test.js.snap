// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main snapshot test 1`] = `
<div>
  <nav
    className="navbar navbar-expand navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Itunes Search
    </a>
    <div
      className="mr-auto navbar-nav"
      onKeyDown={[Function]}
    />
  </nav>
</div>
`;
